mm2src/coins/eth/wallet_connect.rs

```diff
@@ -15,13 +15,17 @@ use kdf_walletconnect::{chain::{WcChainId, WcRequestMethods},
                         error::WalletConnectError,
                         WalletConnectCtx};
 use mm2_err_handle::prelude::*;
+use secp256k1::PublicKey;
 use secp256k1::{recovery::{RecoverableSignature, RecoveryId},
                 Secp256k1};
 use std::str::FromStr;
 use web3::signing::hash_message;
 
 use super::EthCoin;
 
+// Wait for 20 seconds for the transaction to appear on the RPC node.
```

Comment:
this should say 30 seconds. also better have this as a doc comment for LSP.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2223/commits/10e9e677ab9242342b9168061b87546c748089d0